src/Git/GetVersionCollectionFromGitRepository.php

```diff
@@ -37,7 +37,10 @@ public function fromRepository(CheckedOutRepository $checkedOutRepository) : Ver
                 } catch (InvalidVersionStringException $e) {
                     return null;
                 }
-            }, explode("\n", $output))
+            }, explode("\n", $output)),
+            function (?Version $version) : bool {
```

Comment:
This fix seems incorrect: a union type `null|object` is indeed filterable. Is this something new in phpstan?

Reply:
Unsure. Would you prefer the ignore?

Reply:
Well, leaving this out completely should work just fine...

Reply:
https://github.com/phpstan/phpstan/pull/877 is this related?

Reply:
Not sure, but it looks related. `null|object` seems to be indeed something that should land in that patch.